navigation/mapping_exploration/launch/explore_lite.launch

```diff
@@ -0,0 +1,16 @@
+<launch>
+<node pkg="explore_lite" type="explore" respawn="true" name="explore" output="screen">
+  <param name="robot_base_frame" value="create1_tf/base_link"/>
+  <param name="move_base_topic" value="create1/move_base"/>
+  <param name="costmap_topic" value="map"/>
+  <param name="costmap_updates_topic" value="map_updates"/>
```

Comment:
Move all the constant parameters to a YAML file (as it's being done in `move_base` for example) inside a `config` directory.

Reply:
Done! Also wrote some explanation of the parameters in the Yaml